### ISSUE TITLE
Make sure that following the README results in a usable system

### ISSUE THREAD

**Issue by SCdF** [medic/medic-webapp#2498]
This may be resolved when 0.4 configurations work on 2.x, but ideally a random internet developer could follow the medic-webapp readme and end up with a fully working system, including some example forms, users etc. This would aid in onboarding new people, as it would remove the current necessity of pulling in random secret data from undocumented places.

**Comment by nice-snek** [comment]
hi frand @abbyad, please [triage](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues) before the end of this sprint. :parrot_conga::parrot_conga::parrot_conga:

**Comment by abbyad** [comment]
I think we will be closer to this with the instructions for Standard config. Will schedule to make sure that we put this documentation is a useful location for "random internet developers".

**Comment by SCdF** [comment]
@abbyad FWIW unless we open source our standard config (out of interest, would that be such a big deal?) and open source or config deployment tooling (which I think we should definitely do) people off the metaphorical street do not have access to standard.

**Comment by cbrwizard** [comment]
Heya, I'd love to contribute but setting up the forms and tasks is kinda difficult. I'd love to see the tasks as described on this page https://docs.google.com/document/d/1UwSe8H3xykvOUwu65KxJYgBTxv40Y_9b3CPgRbcXOH0/edit#bookmark=id.ta51n5b830h. I've tried using the forms provided in a `demo-forms` folder, but I couldn't fill them in the Reports-Submit report, as there were no inputs displayed.
What's the standard config, can it help? [: Perhaps there is a script somewhere with seed data I can use?

**Comment by SCdF** [comment]
@cbrwizard agreed forms and tasks are terrible externally. This is because historically these two items were partner specific (and so not available for public consumption).

The standard config you mention **is** something that we can expose externally, and is something we should definitely do, as noted in #3779
